src/App.js

```diff
@@ -5,6 +5,8 @@ import TimerPage from './pages/timerPage/timerPage';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import Settings from './pages/Settings/settings-page';
 import KnowMore from './pages/KnowMore/knowmore';
+import Login from './pages/login/login';
+import Signup from './pages/login/signup';
```

Comment:
Could use Barrel exports too making it easy for importing all the pages in a single line itself:
`import {Login,Signup,KnowMore,Settings} from "./pages"`